frontend/src/pages/notebookController/SpawnerPage.tsx

```diff
@@ -76,6 +81,31 @@ const SpawnerPage: React.FC<SpawnerPageProps> = React.memo(({ setStartModalShown
     return EMPTY_USER_STATE;
   }, [dashboardConfig, translatedUsername]);
 
+  const onModalClose = () => {
+    setStartShown(false);
+    if (notebook) {
+      deleteNotebook(projectName, notebook?.metadata.name).catch((e) => console.error(e));
```

Comment:
This error gets swallowed and that could be problematic, no? Should we look to present this error somewhere?